Examples/voc_convert.bat

```diff
@@ -0,0 +1,30 @@
+@echo off
```

Comment:
not sure we want this file in the repo

Reply:
Yeh wasn't sure about that, I didn't realise they could be potentially nasty files. Will remove.

Reply:
I don't think it's nasty, but it's a bit too specific for most users, while something like the README should be reasonably accessible???